bindings/java/README.md

```diff
@@ -1,4 +1,12 @@
-<h1>JUCP</h1>
+<h1>JUCX</h1>
 
-JUCP is a Java API over UCP (UCX protocol).</br>
+JUCX is a Java API over UCP (UCX protocol).</br>
 See more about UCX at: https://github.com/openucx/ucx
+
+# Building JUCX
+
+Building the source requires [Apache Maven](http://maven.apache.org/) and [GNU/autotools](http://www.gnu.org/software/autoconf/autoconf.html) and Java version 8 or higher.</br>
+In order to build, execute the following steps:
+
+1. export JAVA_HOME=<path-to-java>.
+2. When running UCX's "configure" add "--with-java" flag, i.e. "shell$ ./configure --with-java".
```

Comment:
Do we want to enable it by default if java in the path ?

Reply:
@yosefe @miked-mellanox what do you think?

Reply:
+1

Reply:
enable by default, continue if not found

Reply:
So if java and mvn and is in the path --> compile as if with-java was selected, else --> compile without jucx?
Should I change the 'with-java' option to 'without/disable-java'?

Reply:
yes and yes.